src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -196,10 +203,11 @@ namespace Microsoft::Console::VirtualTerminal
         _Success_(return ) bool _VerifyDeviceAttributesParams(_In_reads_(cParams) const unsigned short* const rgusParams,
                                                               const unsigned short cParams) const;
 
-        _Success_(return ) bool _GetPrivateModeParams(_In_reads_(cParams) const unsigned short* const rgusParams,
-                                                      const unsigned short cParams,
-                                                      _Out_writes_(*pcParams) DispatchTypes::PrivateModeParams* const rgPrivateModeParams,
-                                                      _Inout_ size_t* const pcParams) const;
+        template<typename TParamType, bool bIgnoreNarrowingConversionFailures = true>
+        _Success_(return ) bool _GetTypedParams(_In_reads_(cParams) const unsigned short* const rgusParams,
+                                                const unsigned short cParams,
+                                                _Out_writes_(*pcParams) TParamType* const rgXtPushParams,

```

Comment:
Nit: Shouldn't `rgXtPushParams` now be named `rgTypedParams` to match the method definition?

Reply:
Should be resolved now that Niksa's changes got rid of all the pointer/count stuff anyway.